Core/clim-basic/extended-streams/stream-output.lisp

```diff
@@ -350,7 +351,9 @@
                                              :margin margin
                                              :break-strategy break
                                              :start start :end end))
-              do (stream-write-output stream string start split)
+              do (maxf (%stream-char-height stream)
+                       (text-style-height text-style medium))
```

Comment:
Doing this within the loop seems at odds with the comment "Also text-style doesn't change until the end of this function. -- jd 2019-01-10" a few lines above. Given that the text style is stored in a lexical variable and, as far as I can tell, not changed, that comment seems to be right.

Reply:
right, I've bound text-style-height once. fixed.

we need to call maxf repeatedly after each line because we don't want to set the stream-char-height for a line to which we do not write (i.e `"foo\nbar\n[no-further-output]"`), and to do that we need to zero the stream-char-height after each newline.